### PR TITLE
Users/ajya/azurermwebappbugbashfix

### PR DESCRIPTION
**Pull request by Ajay-MS**
Resolved following issues of bug bash

1. Unable to copy/paste the msdeploy.exe command line from task log into powershell and run it.
2. It should print path of website where is being deployed.
3. On providing non existing webapp name , task failed with null exception
4. Help markdown at bottom of task is not valid

### CODE REVIEW

File: Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ps1 [hunk 1]
```diff
@@ -86,4 +86,9 @@ $msDeployCmdArgs = Get-MsDeployCmdArgs -packageFile $packageFile -webAppNameForM
 # Deploy azureRM webApp using msdeploy Command
 Run-MsDeployCommand -msDeployExePath $msDeployExePath -msDeployCmdArgs $msDeployCmdArgs
 
+# Publish azure webApp url
+$azureWebsitePublishURL = Get-WebAppPublishURL -webAppName $WebAppName -deployToSlotFlag $DeployToSlotFlag -slotName $SlotName
+
+Write-Host (Get-LocalizedString -Key "Webapp successfully published at Url : $azureWebsitePublishURL" -ArgumentList $azureWebsitePublishURL)
```
Comment (mvvsubbu) [inline Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ps1]: Add this string to agent.strings

**Comment by mvvsubbu** [comment]
Looks fine. Add Agent.strings before check in

File: Tasks/AzureRmWebAppDeployment/Utility.ps1 [hunk 2]
```diff
@@ -59,6 +59,23 @@ function Get-WebAppNameForMSDeployCmd
...
+function Get-WebAppPublishURL
+{
+    param([String][Parameter(Mandatory=$true)] $webAppName,
+          [String][Parameter(Mandatory=$true)] $deployToSlotFlag,
+          [String][Parameter(Mandatory=$false)] $slotName)
+
+    $WebAppPublishURL = $webAppName
+    if($deployToSlotFlag -eq "true")
+    {
+        $WebAppPublishURL += "-" + $SlotName
```
Comment (kmkumaran) [inline Tasks/AzureRmWebAppDeployment/Utility.ps1]: minor comment - keep the casing same for slotName parameter.

File: Tasks/AzureRmWebAppDeployment/Utility.ps1 [hunk 3]
```diff
@@ -59,6 +59,23 @@ function Get-WebAppNameForMSDeployCmd
...
+    param([String][Parameter(Mandatory=$true)] $webAppName,
+          [String][Parameter(Mandatory=$true)] $deployToSlotFlag,
+          [String][Parameter(Mandatory=$false)] $slotName)
+
+    $WebAppPublishURL = $webAppName
+    if($deployToSlotFlag -eq "true")
+    {
+        $WebAppPublishURL += "-" + $SlotName 
+    }
+    $WebAppPublishURL += ".azurewebsites.net"
```
Comment (kmkumaran) [inline Tasks/AzureRmWebAppDeployment/Utility.ps1]: Is it possible to get this as part of any Get Webapp commands?

**Pull request merged**